core/src/test/java/de/evoila/cf/broker/controller/utils/DashboardAuthenticationRedirectBuilderTest.java

```diff
@@ -0,0 +1,66 @@
+package de.evoila.cf.broker.controller.utils;
+
+import de.evoila.cf.broker.model.DashboardClient;
+import de.evoila.cf.broker.model.catalog.Dashboard;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.net.URISyntaxException;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class DashboardAuthenticationRedirectBuilderTest {
+
+    private DashboardAuthenticationRedirectBuilder builder;
+
+    @Mock
+    private Dashboard dashboard;
+    @Mock
+    private DashboardClient dashboadClient;
+
+    @Test
+    void getRedirectUrl_Valid() throws  URISyntaxException {
+        when(dashboard.getAuthEndpoint()).thenReturn("base");
+        when(dashboadClient.getId()).thenReturn("123456");
+
+        builder = new DashboardAuthenticationRedirectBuilder(dashboard, dashboadClient, "redirekt", "testscope");
```

Comment:
Is there a reason why you call it `redirekt` instead of `redirect` or was it just a typo and got copied&pasted?

Reply:
typo and copy&paste...